WMF Framework/Remote Notifications/RemoteNotificationsOperationsController.swift

```diff
@@ -0,0 +1,178 @@
+class RemoteNotificationsOperationsController {
+    private let apiController: RemoteNotificationsAPIController
+    private let modelController: RemoteNotificationsModelController?
+    private let deadlineController: RemoteNotificationsOperationsDeadlineController?
+
+    private let syncRepeatingTime: Int = 15
```

Comment:
Does this mean it's still checking every 15 seconds? Oh, related, should this polling be reachability aware?

Reply:
(on both counts can be follow-ups as needed)